<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPswd</class>
 <widget class="QWidget" name="ModifyPswd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>672</width>
    <height>428</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>672</width>
    <height>428</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/pic/Settings_1366px_1181299_easyicon.net.png)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>251</width>
     <height>61</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;我的个人设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>651</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>651</width>
     <height>20</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;旧密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;新密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;确认密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;圆体-简&quot;;
border-image: url(:/pic/yes_200px_1188209_easyicon.net.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>330</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;圆体-简&quot;;
border-image: url(:/pic/Cancel_110px_1190738_easyicon.net.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>651</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>651</width>
     <height>21</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>161</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>161</width>
     <height>61</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/wild)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>line_2</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>BackButton</zorder>
  <zorder>CancelButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
